src/lib/device-config.ts

```diff
@@ -42,9 +44,10 @@ export const generateConfig = (
 	const userPromise = getUser(req);
 
 	// Devices running ResinOS >=1.2.1 are capable of using Registry v2, while earlier ones must use v1
-	const registryHost = resinSemver.gte(osVersion, '1.2.1')
-		? REGISTRY2_HOST
-		: REGISTRY_HOST;
+	if (resinSemver.lte(osVersion, '1.2.0')) {
+		throw new BadRequestError('Supervisor API url must be specified');
```

Comment:
Did you copy/paste the error message by chance?

Reply:
@Page- added a `Deprecated ResinOS version` message for review.